src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,15 @@
+@mixin hand($var, $name, $time, $angle, $step) {
```

Comment:
It's better to use more descriptive variable names instead of $var, for example: $hand-type.